### PR TITLE
Retrieve published version of element in event listener

### PR DESCRIPTION
**Pull request by limenet**

### CODE REVIEW

File: src/EventListener/Pimcore/AbstractListener.php [hunk 1]
```diff
@@ -160,4 +161,23 @@ protected function deleteElementFromIndex(AbstractElement $element, IndexInterfa
         $elasticsearchId = $indexDocument->getElasticsearchId($element);
         $index->getElasticaIndex()->deleteById($elasticsearchId);
     }
+
+    /**
+     * The object passed via the event listener may be the a draft and not the latest published version.
```
Comment (pryserv) [inline src/EventListener/Pimcore/AbstractListener.php:166]: remove "the" in "may be the a draft"

File: src/Exception/EventListener/PimcoreElementNotFoundException.php [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+namespace Valantic\ElasticaBridgeBundle\Exception\EventListener;
+
+use Valantic\ElasticaBridgeBundle\Exception\BaseException;
+
+class PimcoreElementNotFoundException extends BaseException
+{
+    public function __construct(?int $id, string $type)
+    {
+        parent::__construct(sprintf('Pimcore element ID %d of type %s not found', $id, $type), 0, null);
```
Comment (pryserv) [inline src/Exception/EventListener/PimcoreElementNotFoundException.php:13]: You don't need to set parent `$code` and `$previous` parameter

**Review by pryserv — APPROVED** [review]

File: src/EventListener/Pimcore/AbstractListener.php (hunk 1 above)
Comment (limenet) [reply src/EventListener/Pimcore/AbstractListener.php:166]: ouch, thanks!

File: src/Exception/EventListener/PimcoreElementNotFoundException.php (hunk 2 above)
Comment (limenet) [reply src/Exception/EventListener/PimcoreElementNotFoundException.php]: Copy-pasted from other exceptions, so I fixed it there as well. Thanks!

**Pull request merged**